### ISSUE TITLE
Accessibility: Social links are missing screen reader support

### ISSUE THREAD

**Issue by JoeProgrammer88** [SpeakingInBits/PC2#202]
```html
<a href="https://twitter.com/PC24DD" target="_blank" > <img src="/images/twitter-logo.jpg" alt="twitter logo"> </a>
<a href="https://www.facebook.com/PCCDD" target="_blank" > <img src="/images/facebook-logo.jpg" alt="facebook logo"> </a>
```
Accessibility checker states: Links that open in a new tab or a new window should either have an "aria-label" attribute or a screen-reader only element explaining to screen-readers that this opens in a new tab.
